### ISSUE TITLE
TextEdit does not work

### ISSUE THREAD

**Issue by goatchurchprime** [aaronfranke/godot-GUI-in-VR#2]
Would this need an full pop-up ASCII keyboard pane to work?  (I don't know if this asset already exists or not.)

If you initialize the TextEdit box with some text, the VR pointer does place the cursor where you want it to go.

Maybe for a starting point, can you take key-strokes from the PC and make them appear in the TextEdit box when it has focus?

**Comment by aaronfranke** [comment]
Yes, getting keyboard input would be a good improvement, or maybe using the built-in popup keyboard in SteamVR/etc. Do you know of a way to get keyboard input to the GUI or to ask SteamVR for the keyboard?

**Comment by marc-weber1** [comment]
I think it's because viewports don't receive input by default: https://godotengine.org/qa/32261/how-to-forward-keyboard-input-to-sub-viewport

I'll try to send a PR if I fix it
